pkg/services/ngalert/sender/notifier.go

```diff
@@ -336,7 +336,7 @@ func (n *Manager) Send(alerts ...*Alert) {
 			}
 		})
 
-		a.Labels = lb.Labels(a.Labels)
+		a.Labels = lb.Labels()
```

Comment:
@grafana/alerting-backend-product -- the prometheus upgrade no longer has this function, but also the previous logic did not make sense.  And I don't see any test :(